src/Application/ApplicationFileProcessor.php

```diff
@@ -148,7 +148,7 @@ public function processFiles(array $files, Configuration $configuration): array
     /**
      * @param string[] $filePaths
      */
-    public function configurePHPStanNodeScopeResolver(array $filePaths, Configuration $configuration): void
+    public function configurePHPStanNodeScopeResolver(array &$filePaths, Configuration $configuration): void
```

Comment:
looks like after this method was called the "correct list" of files should be contained in `$this->nodeScopeResolver->setAnalysedFiles()`.

could the caller of `configurePHPStanNodeScopeResolver` get it from there?

Reply:
I updated to return file paths instead https://github.com/rectorphp/rector-src/pull/4513/commits/9b38ba5ad6adbe9e48a9de62348e808226805057